.changeset/funny-onions-join.md

```diff
@@ -0,0 +1,5 @@
+---
+"@quri/ui": patch
+---
+
+z-index order for dropdown, modal. toasts
```

Comment:
What are these changesets for? I assume the release framework we're using?

Reply:
Yes, https://github.com/changesets/changesets

There's a comment by changeset-bot in this thread that links to the documentation.